appeals/web/src/server/lib/mappers/data/lpa-questionnaire/s78.js

```diff
@@ -33,5 +35,7 @@ export const submaps = {
 	isAonbNationalLandscape: mapIsAonbNationalLandscape,
 	isInfrastructureLevyFormallyAdopted: mapIsInfrastructureLevyFormallyAdopted,
 	infrastructureLevyAdoptedDate: mapInfrastructureLevyAdoptedDate,
-	infrastructureLevyExpectedDate: mapInfrastructureLevyExpectedDate
+	infrastructureLevyExpectedDate: mapInfrastructureLevyExpectedDate,
```

Comment:
can we turn comma dangle back on 😩

Reply:
Is that an eslint config change, or something in github?

Reply:
It just means lines like this already have a comma so the diff is shorter when you add to an object or array

Reply:
It would be nice not to have these in the diff!